webpack.config.server.js

```diff
@@ -51,9 +51,9 @@ module.exports = ({ resolvePath, START_DEV_SERVER }) => {
         patterns: [
           {
             from: 'public/**/*',
-            globOptions: {
-              ignore: ['**/images/**'],
-            },
+            // globOptions: {
+            //   ignore: ['**/images/**'],
+            // },
```

Comment:
Please note that as part of this change the [local comscore route](https://github.com/bbc/simorgh/blob/898b1d15e69018550c5261f346fa40700bb7abb8/src/server/local/index.js#L208-L215)
will need to be removed since it's no longer required. 
`CopyPlugin` duplicates the static files into the desired directory.